scripts/component-base.js

```diff
@@ -12,15 +10,17 @@ import {
   flatAsValue,
   flat,
   mergeUniqueArrays,
-  getBlocksAndGrids,
 } from './libs.js';
 import { externalConfig } from './libs/external-config.js';
+import { generalManipulation, generateDom, renderVirtualDom } from './render/dom.js';
 
 export default class ComponentBase extends HTMLElement {
   // All supported data attributes must be added to observedAttributes
   // The order of observedAttributes is the order in which the values from config are added.
   static observedAttributes = [];
 
+  dataAttributesKeys = [];
```

Comment:
This should be moved in the setDefaults() method for consistency

Reply:
In my tests in the begining that worked as it was before, we could change from an element directly any of these properties. 

And for `config` and `nestedComponentsConfig` which we are extending in components keeping the defaults from base, the extension failed if these were class fields.